electron-builder.yml

```diff
@@ -1,5 +1,6 @@
 productName: Rancher Desktop
 icon: ./resources/icons/logo-square-512.png
+appId: io.rancher.rd
```

Comment:
Instead of `rd` can we make it something like `rancher-desktop`. The repo name being rd was so it was not obvious. We may end up renaming the repo at some point.

Reply:
Changed in ef10790baba085744bd76ff5bffa7d7a40609874 — this is arbitrary anyway.